cfgov/v1/views/reports.py

```diff
@@ -101,8 +106,48 @@ class DocumentsReportView(ReportView):
     template_name = "v1/documents_report.html"
 
     def get_filename(self):
-        """Get a better filename than the default 'spreadsheet-export'."""
-        return f"all-cfgov-documents-{date.today()}"
+        return generate_filename("documents")
 
     def get_queryset(self):
         return Document.objects.all().order_by("-id").prefetch_related("tags")
+
+
+class ImagesReportView(ReportView):
+    header_icon = "image"
+    title = "All images"
+
+    list_export = [
+        "title",
+        "file.url",
+        "file_size",
+        "collection",
+        "tags.names",
+        "created_at",
+        "uploaded_by_user",
+    ]
+    export_headings = {
+        "title": "Title",
+        "file.url": "File",
+        "file_size": "Size",
+        "collection": "Collection",
+        "tags.names": "Tags",
+        "created_at": "Uploaded on",
+        "uploaded_by_user": "Uploaded by",
+    }
+
+    custom_field_preprocess = {
+        "tags.names": {"csv": process_tags},
+    }
+
+    template_name = "v1/images_report.html"
+
+    def get_filename(self):
+        return generate_filename("images")
+
+    def get_queryset(self):
+        return (
+            Image()
```

Comment:
See above; someone reading this might interpret it as creating a new instance of the image class, which would be confusing.

```suggestion
            get_image_model()
```